frontend/.storybook/config.js

```diff
@@ -1,4 +1,5 @@
 import { configure } from '@storybook/react';
+import '../src/index.css';
```

Comment:
So we do this instead of the preview-head.html thing, right?

Reply:
Yep! mostly because I couldn't figure out the right path to throw in the link tag.